### DISCUSSION TITLE
rootfs AB reduncancy and OTA status on JP5 kirkstone branch

### DISCUSSION THREAD

**Discussion by GFallasRR** [OE4T/meta-tegra#1272] [Q&A]
Hi all,

Just wondering the status of the OTA support and the rootfs AB redundancy on the JP5 kirkstone branch for Orin AGX, I found some of the tools used for OTA and the slots controls but only bootloader slots seem to be enabled, for the rootfsAB there is a message that the  RootFS A/B is not enabled when using nvbootcontrol. Is there a way to enable it or is it not supported yet?

Thanks.

**Comment by kekiefer** [comment]
It works in JP5, but unfortunately it's still a DIY effort without much documentation or examples . The implementation ties the bootloader A/B slots to the rootfs A/B. You can see https://github.com/OE4T/meta-tegra/discussions/1182#discussioncomment-5067058 for some information on how to enable.

**Comment by dwalkes** [comment]
See also the discussion in the https://matrix.to/#/#OE4T_jetpack-5-device-update:gitter.im for latest Jetpack 5 device update work.

**Comment by dwalkes** [comment]
This was also a topic of discussion in the meeting notes this week - see https://github.com/OE4T/meta-tegra/wiki/OE4T-Meeting-Notes-2023-06-08 and linked video
